pkgs/development/libraries/tdlib/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, gperf, openssl, readline, zlib }:
+
+stdenv.mkDerivation rec {
+  version = "2018-03-20";
+  name = "tdlib-${version}";
+
+  src = fetchFromGitHub {
+    owner = "tdlib";
+    repo = "td";
+    rev = "cfe4d9bdcee9305632eb228a46a95407d05b5c7a";
```

Comment:
Furthermore, seems weird that they themselves have tagged it as [v1.2.0](https://github.com/tdlib/td/releases/tag/v1.2.0) which probably should be the version instead of the date.